datus/models/litellm_adapter.py

```
@@ -0,0 +1,256 @@
+    """
+
+    # Model name prefix mapping for LiteLLM
+    # See: https://docs.litellm.ai/docs/providers
+    MODEL_PREFIX_MAP = {
+        "openai": "",  # LiteLLM supports OpenAI models natively without prefix
+        "claude": "anthropic/",
+        "deepseek": "deepseek/",
+        "qwen": "dashscope/",
+        "gemini": "gemini/",
+        "kimi": "moonshot/",  # Moonshot AI - https://docs.litellm.ai/docs/providers/moonshot
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)